app/controllers/transmuxer/notifications_controller.rb

```diff
@@ -20,4 +20,4 @@ def handle_job_notification
       resource.update_transmuxer_job(state: params[:job][:state], metadata: params[:input])
     end
   end
-end
\ No newline at end of file
+end
```

Comment:
I'm being a bit pedantic but imo, this change, though valid, should not be part of the current PR.